compose_files/keycloak/realm.json

```diff
@@ -26,7 +26,7 @@
   "oauth2DeviceCodeLifespan": 600,
   "oauth2DevicePollingInterval": 5,
   "enabled": true,
-  "sslRequired": "external",
+  "sslRequired": "none",
```

Comment:
what does sslRequired: none do? why wouldn't you want ssl? aren't the urls https ?

Reply:
For the automated tests ssl isn't needed and just adds complexity. This may not be required with the change to start-dev but I want to to eliminate all posibilities of an HTTPs redirect for testing.